### DISCUSSION TITLE
recreate missing pvc's?

### DISCUSSION THREAD

**Discussion by smiyc** [cloudnative-pg/cloudnative-pg#2006] [Ideas]
follow up discussion from https://cloudnativepg.slack.com/archives/C03AX0J5P29/p1683529308328529

during some test's, I deleted the "-join- pods" pvc's and after that the pod got stuck with:
Warning  FailedScheduling  80s (x1 over 2m20s)  default-scheduler  0/8 nodes are available: 8 persistentvolumeclaim "keycloak-test-6" not found.

This could be fixed by deleting the -join- JOB.

I don't know if this will break something else, but from my point of view, if the operator recognice that the pod pvc's are missing, they could be recreated.

**Comment by Pivert** [comment]
Same problem. 
I destroyed a replica when realizing I had 2 on the same node, but it did not re-create PVCs.
Not sure if I should manually provide a pvc ?
